lib/config/presets/internal/group.ts

```diff
@@ -698,6 +698,15 @@ const staticGroups = {
       },
     ],
   },
+  vaadinJavascript: {
+    description: 'Group all `@vaadin` packages together.',
+    packageRules: [
+      {
+        groupName: 'Vaadin javascript packages',
+        matchPackagePrefixes: ['@vaadin/'],
```

Comment:
can't they grouped as monorepo via sourceUrl?

Reply:
It's probably gonna be better this way, since it will only match the web components and not everything under `@vaadin`.